lib/workflow/client.js

```diff
@@ -140,6 +140,7 @@ class Workflow {
           const task = response.json.data;
           return this.client(`/${taskId}`, {
             method: 'PUT',
+            query: { preserveUpdatedAt: true },
```

Comment:
I think the `extend` method immediately above will also want this same treatment.

Reply:
👍 Have added the parameter to that call now too.